app/scripts/modules/core/src/managed/NoticeCard.tsx

```diff
@@ -3,14 +3,15 @@ import classNames from 'classnames';
 import styles from './NoticeCard.module.css';
 
 interface INoticeCardProps {
-  title: string;
-  text: string;
+  title: JSX.Element | string;
+  text: JSX.Element | string;
   icon: string;
-  noticeType: 'ok'; // TypeScript doesn't like this being arbitrary strings because it can't validate styles[noticeType]
+  noticeType: 'success' | 'neutral' | 'info' | 'error';
```

Comment:
i ended up needing to do a bit more refactoring on `NoticeCard` in this PR than I had anticipated/hoped as there's a wider variety of states this component needed to support. I'm holding off on doing a full refactor of it as Greg is in the process of tightening up the markup/states.